src/ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj

```diff
@@ -16,17 +16,29 @@
     <PackageReference Include="Aspire.Azure.Storage.Queues" />
     <PackageReference Include="Azure.Extensions.AspNetCore.Configuration.Secrets" />
     <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="FluentValidation.AspNetCore" />
+    <PackageReference Include="Microsoft.AspNetCore.ApiPagination" />
+    <PackageReference Include="Microsoft.AspNetCore.ApiVersioning" />
+    <PackageReference Include="Microsoft.AspNetCore.ApiVersioning.Swashbuckle" />
+    <PackageReference Include="Microsoft.AspNetCore.WebHooks.Receivers" />
+    <PackageReference Include="Microsoft.AspNetCore.WebHooks.Receivers.GitHub" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
     <PackageReference Include="Microsoft.Identity.Web" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" />
     <PackageReference Include="Swashbuckle.AspNetCore" />
+    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" />
+    <PackageReference Include="Swashbuckle.AspNetCore.Newtonsoft" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" VersionOverride="7.0.0" />
```

Comment:
Should we maybe leave a comment for why the version is overriden?

Reply:
I wanted to do that but if you remove it, the compiler will actually tell you exactly what's the dependency chain and it will always stay correct as apposed to us keeping comments around that get outdated fast.